### DISCUSSION TITLE
How to enter exceptional meeting dates?

### DISCUSSION THREAD

**Discussion by stepstra** [sircharlo/meeting-media-manager#1622] [Q&A] [answered]
Usually a KH is shared by two or more congregations. When one has their CO visit, the one that meets on Tuesdays has to change their meeting day. How can that be achieved in M³?

**Comment by e-Feitosa** [comment, marked as answer]
The M3 already has this option for the SC visit, however, if there is a congregation that meets on Tuesdays, he needs to make the change manually. Only the congregation receiving the visit has the option to change whether to include the date of the visit. See the print where you configure the date of the visit.

![image](https://user-images.githubusercontent.com/90984283/223519037-dee8970b-55cb-4f32-a844-c0961d81c728.png)

**Comment by stepstra** [comment]
So I should change the regular date within the week where it occurs and the next week after that back to normal?

**Reply by e-Feitosa** [reply]
You can enter the visit date at any time.

**Reply by stepstra** [reply]
You misunderstood. We are the congregation that has to move to wednesday in this week, because another congregation has their visit.

**Reply by e-Feitosa** [reply]
Yes, I understood, in the first answer I mentioned that the congregation that already gathers on Tuesday, needs to change the date manually, and in this case, you do not have the option to make the advance change. It was not thought of this automatic change option if a congregation already has a meeting on Tuesday. However, you can change the date of the meeting at any time in the settings if you want to check if the media are all right.

**Comment by e-Feitosa** [comment]
@mtdvlpr, did you think of something about this situation above?

**Reply by mtdvlpr** [reply]
Not really. Like you said, you can manually change the meeting day. To ensure other brothers use the new meeting day, you can lock it through the congregation sync server. I think adding another date select with a weekday select would be more trouble than it's worth. Especially since this situation would occur twice a year only.

**Reply by stepstra** [reply]
Actually, it occurs four times a year for us, since there are two other congregations in the hall. ;-)

But don't get me wrong - that's alright, I just wanted to know, if changing and locking the meeting day via sync is the correct way to go.

And if there is a way to enter that in advance, but as @e-Feitosa pointed out there is none. So I have to do it on Monday of the corresponding week, change meeting day to Wednesday, lock it in the sync and after Wednesday and I can reverse it, right?

**Reply by mtdvlpr** [reply]
Tell you what, if the fallback meeting day is (almost) always the same, we could add a third weekday select (only visible when your midweek meeting day is Tuesday). Then we could add some logic that checks the CO week value of the other congregations and automatically set the midweek meeting for that week to the fallback day.

**Reply by stepstra** [reply]
I fear that will complicate things. You see, we meet on Tuesdays, the Polish cong. Wednesdays and the Spanish Thursdays. This means, we would have to exchange the meeting day with the according congregation. Sometimes Wednesday, sometimes Thursday. Someday someone might even come along and say: hey, why not use Friday? :-P

**Reply by mtdvlpr** [reply]
In that case it won't make life simpler. Only change the option that needs to be altered everytime...
